### ISSUE TITLE
Support searching recipes

### ISSUE THREAD

**Issue by marco-c** [mozilla/serviceworker-cookbook#112]
Once we add more metadata, a search functionality would be really useful.

**Comment by darkwing** [comment]
Would this just search content or code as well?   I would think the category name + recipe would be sufficient.

**Comment by marco-c** [comment]
Yeah I'd say only content. We could also add tags which specify the APIs used by a particular recipe, so users can search by API.

**Comment by digitarald** [comment]
Searching code would be nice though, "as a user I want to see all examples
using 'claim'.".

On Thursday, November 19, 2015, Marco <notifications@github.com> wrote:

> Yeah I'd say only content. We could also add tags which specify the APIs
> used by a particular recipe, so users can search by API.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/serviceworker-cookbook/issues/112#issuecomment-158161514>
> .
>


-- 
Web Craftsman | harald@mozilla.com | @digitarald [skype/irc/line] |
+1-650-963-6612 [wapp/telegram]
